waf build error: error: no member named 'to_string'

Hey, seeing the following errors/warnings while trying to build latest cquery.
```
[34/88] Compiling src/messages/cquery_file_info.cc
../../src/messages/cquery_member_hierarchy.cc:113:36: error: no member named 'to_string' in 'std::__1::basic_string_view<char, std::__1::char_traits<char> >'
    entry->name = def->ShortName().to_string();
                  ~~~~~~~~~~~~~~~~ ^
../../src/messages/cquery_member_hierarchy.cc:180:45: error: no member named 'to_string' in 'std::__1::basic_string_view<char, std::__1::char_traits<char> >'
      entry.name = def->DetailedName(false).to_string();
                   ~~~~~~~~~~~~~~~~~~~~~~~~ ^
2 errors generated.

../../src/messages/text_document_completion.cc:30:1: warning: unused function 'Reflect' [-Wunused-function]
MAKE_REFLECT_TYPE_PROXY(lsCompletionTriggerKind);
^
../../src/serializer.h:86:3: note: expanded from macro 'MAKE_REFLECT_TYPE_PROXY'
  MAKE_REFLECT_TYPE_PROXY2(type_name, std::underlying_type<type_name>::type)
  ^
../../src/serializer.h:93:15: note: expanded from macro 'MAKE_REFLECT_TYPE_PROXY2'
  inline void Reflect(Writer& visitor, type& value) {                  \
              ^
1 warning generated.

../../src/messages/initialize.cc:108:1: warning: unused function 'Reflect' [-Wunused-function]
MAKE_REFLECT_TYPE_PROXY(lsTextDocumentSyncKind)
^
../../src/serializer.h:86:3: note: expanded from macro 'MAKE_REFLECT_TYPE_PROXY'
  MAKE_REFLECT_TYPE_PROXY2(type_name, std::underlying_type<type_name>::type)
  ^
../../src/serializer.h:88:15: note: expanded from macro 'MAKE_REFLECT_TYPE_PROXY2'
  inline void Reflect(Reader& visitor, type& value) {                  \
              ^
1 warning generated.

Waf: Leaving directory `/Users/nenchev/Developer/tools/cquery/build/release'
Build failed
 -> task in 'bin/cquery' failed with exit status 1 (run with -v to display more information)
cquery $ 
```

--- a/src/messages/cquery_member_hierarchy.cc
+++ b/src/messages/cquery_member_hierarchy.cc
@@ -110,7 +110,7 @@ bool Expand(MessageHandler* m,
   if (detailed_name)
     entry->name = def->detailed_name;
   else
-    entry->name = def->ShortName().to_string();
+    entry->name = std::string(def->ShortName());
   std::unordered_set<Usr> seen;
   if (levels > 0) {
     std::vector<const QueryType*> stack;
@@ -177,7 +177,7 @@ struct Handler_CqueryMemberHierarchy
     Out_CqueryMemberHierarchy::Entry entry;
     // Not type, |id| is invalid.
     if (detailed_name)
-      entry.name = def->DetailedName(false).to_string();
+      entry.name = std::string(def->DetailedName(false));
     else
       entry.name = std::string(def->ShortName());
     if (def->spell) {

